auth-middleware-test-svc/flaskapp.py

```diff
@@ -12,8 +12,16 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+# TODO manage the import when commit and push
```

Comment:
I think these comments can be removed, right?

Reply:
removed